### DISCUSSION TITLE
Avoid rebuild in dev mode on unchanged local images with the Image plugin?

### DISCUSSION THREAD

**Discussion by brycewray** [11ty/eleventy#1887] [Q&A]
*Posting this here as well as in the Eleventy Image “Issues” section ([#108](https://github.com/11ty/eleventy-img/issues/108)) seeking help&nbsp;.&nbsp;.&nbsp;.*

---

When in development mode (using Eleventy with the `--serve` flag) and with only local images, is there a flag (for either the Image plugin or Eleventy itself) that will tell the Eleventy Image plugin **not** to rebuild all the images every time there's a change in the repo **other than** one of the images?

I have read the plugin documentation and every related article I can find, and have yet to see a mention of how to make Image rebuild only a changed local image in dev mode.

Thanks in advance for any guidance anyone can provide about this.

**Comment by noelforte** [comment]
Stumbled across this while perusing the discussions, for anyone else that happens upon this thread, this feature has been implemented in Eleventy Image 1.0: https://github.com/11ty/eleventy-img/issues/51#issuecomment-907930489

**Reply by brycewray** [reply]
True, but I haven't been able to make it work yet; each new build also rebuilds each image, with no observed activity in `/.cache`. Perhaps my setup isn't correct.

**Reply by noelforte** [reply]
hm interesting, right before I posted that I tried running 11ty and checking the modification times of the images in my build directory. timestamps stay the same on my end but i'll have to double-check. local images, right?

**Reply by brycewray** [reply]
Yes, local only. When I've used remote images with this plugin, the caching has always worked perfectly. I've just had no luck with local images, even after the release of v.1.0.

**Reply by minttoothpick** [reply]
I don't think eleventy-img v1.0.0 has actually been released yet; I'm seeing [v0.10.0 as the latest release](https://github.com/11ty/eleventy-img/releases/tag/v0.10.0). I got confused by the comment in [11ty/eleventy-img#51](https://github.com/11ty/eleventy-img/issues/51#issuecomment-907930489) as well.

However, I am using [this fork](https://github.com/zeroby0/eleventy-img/tree/cache) as described in the instructions on [zeroby0/netlify-plugin-11ty](https://github.com/zeroby0/netlify-plugin-11ty): 

```
npm r @11ty/eleventy-img
npm i zeroby0/eleventy-img#cache
```

and my images are now being cached locally.

**Reply by brycewray** [reply]
@minttoothpick Yes, was referring to the beta. Sorry I wasn't clear on that.

**Reply by minttoothpick** [reply]
Whoops, my bad!
